css/style.css

```diff
@@ -54,6 +62,150 @@ body {
   margin: auto;
 }
 
+main,
+header,
+article,
+aside,
+footer,
+section {
+  padding: 5px;
+  float: left;
+}
+
+header {
+  width:100%;
+  background-color: #033048;
+  padding: 40px;
+}
+
+
+
+nav {
+  float: right;
+  font-size: 24px;
+  font-weight: 600;
+  color: yellow;
+  padding: 10px;
+}
+
+
+main {
+  width: 710px;
+  overflow: auto;
```

Comment:
you can remove the `overflow: auto`. Its actually creating a horizontal scroll that you probably don't want